react/src/lib/components/WellCompletions/components/WellCompletionComponent.tsx

```diff
@@ -7,7 +7,8 @@ import DataProvider from "./DataLoader";
 import WellCompletionsViewer from "./WellCompletionsViewer";
 
 // eslint-disable-next-line @typescript-eslint/no-var-requires
-const inputSchema = require("../../../../../inputSchema/wellCompletions.json");
+const inputSchema = require("../../../inputSchema/wellCompletions.json");
```

Comment:
I couldn't find any other way to access this file both from webpack and when using the package as a library package.
When building an npm package, we want to include only the library part. Thus, inputSchema has to be at that level.

Note:
* in an ideal world, the demo and the library part should be in separate packages
* if there is a better solution I'll be happy to apply it, but I couldn't find any